zebra-chain/src/transaction/arbitrary.rs

```diff
@@ -177,6 +186,16 @@ impl Arbitrary for LockTime {
     type Strategy = BoxedStrategy<Self>;
 }
 
+impl Arbitrary for ConsensusBranchId {
```

Comment:
If we switched to NetworkUpgrade, we'd just use an Arbitrary NetworkUpgrade instead.